x/evm/types/params.go

```diff
@@ -56,6 +50,18 @@ func NewParams(evmDenom string, enableCreate, enableCall bool, config ChainConfi
 	}
 }
 
+// ParamSetPairs returns the parameter set pairs.
+func (p *Params) ParamSetPairs() paramtypes.ParamSetPairs {
```

Comment:
isnt this deprecated?

Reply:
@ramacarlucho Those are still needed to make a migration. I've added that function to a new `params_legacy.go` file